appshell/cefclient_mac.mm

```diff
@@ -334,8 +334,15 @@ - (void)windowWillEnterFullScreen:(NSNotification *)notification {
     }
     if (customTitlebar) {
         [customTitlebar setHidden:YES];
+        NSWindow *window = [notification object];
+
+        // Since we have nuked the title, we will have
+        // to set the string back as we are hiding the
+        // custom title bar.
+        [window setTitle:[customTitlebar titleString]];
```

Comment:
Which SDK are you on? I have built this using 10.11 sdk and it seems to work fine.
![screen shot 2016-05-12 at 11 22 00 pm](https://cloud.githubusercontent.com/assets/1577184/15224853/44ecb156-1899-11e6-8f0f-9772a3f5cd83.png)